Test failure in ydb/tests/compatibility/test_user_management.py.TestUserManagementRollingUpgrade.test_user_management[rolling_stable-25-4-1_to_prestable-26-1]

ydb/tests/compatibility/test_user_management.py.TestUserManagementRollingUpgrade.test_user_management[rolling_stable-25-4-1_to_prestable-26-1]

**Test Failure**

**Build Information:**
- Version: 8ea7ec920e76164ab06fa8c5819d5daeea7508ba
- Build Preset: relwithdebinfo
- Branch: main

**Test Report:** [View report](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21044181712/ya-main-x86-64/try_1/ya-test.html#FAIL)

**Run Command:**
```
./ya make -ttt --build "relwithdebinfo" -F 'test_user_management.py::TestUserManagementRollingUpgrade::test_user_management[rolling_stable-25-4-1_to_prestable-26-1]' ydb/tests/compatibility
```

**Owner:** [TEAM:@ydb-platform/engineering](https://github.com/orgs/ydb-platform/teams/engineering)

**Success rate (last 4 days):**
MAIN: 0% (0/9)

**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?branch=main&full_name=ydb/tests/compatibility/test_user_management.py.TestUserManagementRollingUpgrade.test_user_management[rolling_stable-25-4-1_to_prestable-26-1]&build_type=relwithdebinfo)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

**Logs:**
- [log](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21044181712/ya-main-x86-64/try_1/artifacts/logs/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_user_management.py.TestUserManagementRollingUpgrade.test_user_management.rolling_stable-25-4-1_to_prestable-26-1.log)
- [logsdir](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21044181712/ya-main-x86-64/try_1/test_artifacts/ydb-tests-compatibility.zip)

**Error Summary:**
```
ydb/tests/compatibility/test_user_management.py:50: in test_user_management
    session_pool.execute_with_retries(query)
contrib/python/ydb/py3/ydb/query/pool.py:252: in execute_with_retries
    return retry_operation_sync(wrapped_callee, retry_settings)
contrib/python/ydb/py3/ydb/retries.py:137: in retry_operation_sync
    for next_opt in opt_generator:
contrib/python/ydb/py3/ydb/retries.py:98: in retry_operation_impl
    result = YdbRetryOperationFinalResult(callee(*args, **kwargs))
contrib/python/ydb/py3/ydb/query/pool.py:250: in wrapped_callee
    return [result_set for result_set in it]
contrib/python/ydb/py3/ydb/_utilities.py:173: in __next__
    return self._next()
contrib/python/ydb/py3/ydb/_utilities.py:164: in _next
    res = self.wrapper(next(self.it))
contrib/python/ydb/py3/ydb/query/session.py:390: in <lambda>
    lambda resp: base.wrap_execute_query_response(
contrib/python/ydb/py3/ydb/query/base.py:211: in decorator
    return func(rpc_state, response_pb, session_state, *args, **kwargs)
contrib/python/ydb/py3/ydb/query/base.py:229: in wrap_execute_query_response
    issues._process_response(response_pb)
contrib/python/ydb/py3/ydb/issues.py:237: in _process_response
    raise exc_obj(_format_response(response_proto), response_proto.issues)
E   ydb.issues.Unauthorized: message: "Executing ESchemeOpAlterLogin" issue_code: 2018 severity: 1 issues { message: "Access denied for scheme request" issue_code: 2018 severity: 1 issues { message: "Access denied." severity: 1 } } (server_code: 400020)
```

--- a/ydb/tests/compatibility/test_user_management.py
+++ b/ydb/tests/compatibility/test_user_management.py
@@ -10,8 +10,11 @@ class TestUserManagementRollingUpgrade(RollingUpgradeAndDowngradeFixture):
         yield from self.setup_cluster()
 
     def check_login(self, username, password):
-        driver_config = self.driver._driver_config
-        driver_config.credentials = ydb.StaticCredentials.from_user_password(username, password)
+        driver_config = ydb.DriverConfig(
+            database=self.database_path,
+            endpoint=self.endpoints[0],
+            credentials=ydb.StaticCredentials.from_user_password(username, password),
+        )
 
         with ydb.Driver(driver_config) as test_driver:
             with ydb.QuerySessionPool(test_driver) as session_pool:

